src/pages/index.js

```diff
@@ -152,6 +152,7 @@ export default ({ pageContext: { featuredArticles } }) => {
                                         twitchVideo.thumbnailUrl
                                     )}
                                     maxWidth={MEDIA_WIDTH}
+                                    title={twitchVideo.title}
```

Comment:
When did this get removed, and was it on purpose? I know we had some design changes before launch that might have included removing the title

Reply:
This was removed by accident about a week ago when I added video props to the `Card` component. I removed the `Thumbnail` component from the index page in lieu of just using `Card` here instead at that point. [This is the PR](https://github.com/mongodb/devhub/pull/357/files) where that change happened.